stories.json

```diff
@@ -33,5 +33,60 @@
     "permalink": "good-times",
     "text": "We have a holistic view of people. Work life, a family life,  includeanything life are not silos. People have one life. Organizations need to consider that. It is less about partying and onsites (even if they mean a lot to me <3), it is more about having a purpose in your daily work (no bullshit jobs) and the `go` to self-organize. It is about no boundaries to thinking and getting the information you need to make decisions (transparency) and decide as an adult person what is best for you. It is about the flexibility to integrate all elements of life into your day.",
     "background": "northernlights"
+  },
+  {
+    "permalink": "long-run",
+    "text": "We sold a few companies. We are having fun. How do we make it stay that way, how do we make this work?",
+    "background": "gradient"
+  },
+  {
+    "permalink": "public-questions",
+    "text": "I find, in general, the openness of the Slack channel and not directing questions to one individual who you might feel `safe` asking but going right into forum mode. But not only that, the fact that there is a great openness to answering these questions quickly - because of the forum, not the individual.",
+    "background": "bokehballs.002"
+  },
+  {
+    "permalink": "bad-times",
+    "text": "The week before Christmas 2015 the company ran out of money. This was a difficult time, but many who stayed with us then are still with us today. Also, we did ultimately make payroll that month.",
+    "background": "gradient"
+  },
+  {
+  "permalink": "authority-figures",
+  "text": "Timo and Puja publicly encouraging others to question their statements and call bullshit on them where applicable.",
+  "background": "bokehballs.001"
+  },
+  {
+  "permalink": "get-job-done",
+  "text": "It's all about getting the job done. Not when and where - just get it done. Our culture allows for that because we trust each other.",
+  "background": "northernlights"
+  },
+  {
+  "permalink": "acceptable-customers",
+  "text": "We said no to Raytheon (a weapons manufacturer) when they came by our booth and were ready to come back later to sign a $1M contract.",
+  "background": "bokehballs.002"
+  },
+  {
+  "permalink": "common-sense",
```

Comment:
there is already a story with the same 'permalink' - https://culture.giantswarm.io/#common-sense.
This field should be unique, maybe 'common-sense2' ahaha

Reply:
changed the permalink according to your suggestion